roles/ceph-docker-common/tasks/fetch_configs.yml

```diff
@@ -9,6 +9,7 @@
       - /var/lib/ceph/bootstrap-osd/{{ cluster }}.keyring
       - /var/lib/ceph/bootstrap-rgw/{{ cluster }}.keyring
       - /var/lib/ceph/bootstrap-mds/{{ cluster }}.keyring
+      - /var/lib/ceph/bootstrap-rbd-mirror/{{ cluster }}.keyring
```

Comment:
is this key created by `ceph-create-keys`, I haven't found any reference of that key anywhere, not even in `ceph-create-keys`